core/templates/pages/welcome-guide-page/welcome-guide-page.directive.html

```diff
@@ -0,0 +1,79 @@
+<div style="height: auto; background-color: #FFFFFF;" class="row content">
+  <div  class="col-lg-7">
+    <div class="oppia-welcome-guide-header">
+      <h1 class="oppia-welcome-guide-h1" translate="I18N_WELCOME_GUIDE_TITLE"> </h1>
+      <p class = "oppia-welcome-guide-p" translate="I18N_WELCOME_GUIDE_PARAGRAH_1">
```

Comment:
Wanted to check how different these h1 and p styles are from oppia's general h1 and p texts?

Reply:
Remove the space before and after `=`

@Hudda do we have any lint check for these (See other comments in this file)?

Reply:
@DubeySandeep no, we do not have checks for these. Some of these are already added in #8423. I will add others.

Reply:
@DubeySandeep these classes include padding that the original styles do not